pom.xml

```diff
@@ -237,6 +237,19 @@
     <error-prone.version>2.24.0</error-prone.version>
     <error-prone-support.version>0.14.0</error-prone-support.version>
     <doxia.version>1.12.0</doxia.version>
+    <error-prone.configuration-args>
+      <!--
+      Error Prone Support configuration:
+      -->
+      -Xep:JUnitClassModifiers:OFF
+      -Xep:JUnitMethodDeclaration:OFF
+      <!-- Check is not configurable yet. -->
+      -Xep:LexicographicalAnnotationListing:OFF
+      <!-- Too many false positives. -->
+      -Xep:MethodReferenceUsage:OFF
+      -Xep:JUnitValueSource:OFF
+      -Xep:StaticImport:OFF
```

Comment:
> Too many false positives.

better to say `Disabled as we do not see value in such rules`.
Rule might work well, but our team does think it matters or do code less explicit.

Reply:
I pushed a proposal, let me know if you agree to group the checks like this.